bsuite/utils/wrappers_test.py

```diff
@@ -137,5 +138,33 @@ def make_action_sequence(self):
       yield rng.choice(actions)
 
 
+class ReverseGymWrapperTest(absltest.TestCase):
+
+  def test_gym_cartpole(self):
+    self.gym_test_env = gym_wrapper.ReverseGymWrapper(gym.make('CartPole-v0'))
```

Comment:
There's no need to assign anything to `self` here. Can we just have a local `gym_env` variable?